latex2png

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I wasn't aware that the BSDs/NixOS didn't have `bash` in `/bin`! From some cursory googling, it looks like all of the OSes I'd like to support have `/usr/bin/env`, so that sounds good to me.